webapp-django/crashstats/supersearch/form_fields.py

```diff
@@ -180,6 +181,7 @@ def to_python(self, value):
         """
         if value is None:
             return None
-        if str(value).lower() in ('__true__', 'true', 't', '1', 'y', 'yes'):
+        truthy_strings = ('__true__', 'true', 't', '1', 'y', 'yes')
```

Comment:
Yes and no. There's no reuse of it anywhere so the gain of moving it 180ish lines away from where it's being used is only making the code slightly harder to read at the moment. 
Right now, I think the code is pretty easy to read.